go.mod

```diff
@@ -1,6 +1,8 @@
 module knative.dev/security-guard
 
-go 1.18
+go 1.21
+
+toolchain go1.21.4
```

Comment:
Can you remove this line and see what it does? The go-licenses program is broken.

Reply:
@upodroid , 
No success.

Reply:
It worked, you have a broken e2e test that is unrelated to the build job